src/style.css

```diff
@@ -0,0 +1,82 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir.ttc);
+}
+
+.bigger_form {
+  position: relative;
+  margin: 20px 0;
+  height: 70px;
+  border-radius: 4px;
+}
+
+.image_big {
+  height: 19px;
+  width: 19px;
+  position: absolute;
+  left: 26px;
+  top: 25px;
+}
+
+.big_search {
+  padding-left: 62px;
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.smaller_form {
+  position: relative;
+  height: 42px;
+  border-radius: 4px;
+}
+
+.image_small {
+  height: 11px;
+  width: 11px;
+  position: absolute;
+  left: 13px;
+  top: 15px;
+}
+
+.small_search {
+  width: 100%;
+  height: 100%;
+  padding-left: 33px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  font-family: "Avenir", sans-serif;
+  font-size: 14px;
+  line-height: 100%;
+  font-weight: 400;
+}
+
+.bigger_form:hover,
+.smaller_form:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+}
+
+.big_search:focus,
+.small_search:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  outline: none;
+}
+
+.bigger_form:focus,
+.smaller_form:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  outline: none;
+}
```

Comment:
they are the same rules 😉
and do you really need a `:focus` for forms? 😁

Reply:
I had 'too much focusin' ahah 
so sorry, already edited :)